for_or_against.py

```diff
@@ -236,21 +190,26 @@ def analyze_transcript_stances(transcript_path: str, limit: Optional[int] = None
     return analysis_output
 
 
-def save_analysis(analysis: AnalysisOutput, output_path: str = "stance_analysis_results.json"):
-    """Save analysis results to a JSON file."""
-    with open(output_path, 'w') as f:
-        json.dump(analysis.model_dump(), f, indent=2)
-
-
 @click.command()
 @click.option('--transcript', default='transcript.json', help='Path to transcript JSON file')
 @click.option('--limit', type=int, help='Number of speakers to analyze')
-@click.option('--output', default='stance_analysis_results.json', help='Output JSON file path')
+@click.option('--output', default=None, help='Output JSON file path')
```

Comment:
Modified so I could test different copies of this file and have better json output names (that reflect the actual file name)